### ISSUE TITLE
All workers concentrated at 1 place + Pokemon Found Count Bug

### ISSUE THREAD

**Issue by IAmWaseem** [Noctem/pokeminer#44]
![image](https://cloud.githubusercontent.com/assets/1776751/22175033/05e12f80-e00f-11e6-9282-8a5c7f1586ba.png)
see all of my workers are just scanning the same area. And its been like this for a long time

![image](https://cloud.githubusercontent.com/assets/1776751/22175039/2e100e0e-e00f-11e6-8b6a-f62d20f3785d.png)
And the Pokemon Found count is increasing for no reason. It increases by a constant amount every 20 seconds almost.

And workers are insanely slow even though the speed i've set is 22.5

**Comment by m3g4mInD** [comment]
Try    python3 scan.py --bootstarp    @IAmWaseem

**Comment by IAmWaseem** [comment]
@m3g4mInD i'm running using --no-pickle for now since picks mysteries has a bug where the workers go out of the boundary. Let me try adding bootstrap as well

**Comment by m3g4mInD** [comment]
Yes add bootstrap and let map running 1hour and check if all goes nice :)

**Comment by IAmWaseem** [comment]
Still the same :/

**Comment by m3g4mInD** [comment]
Try this values

STAY_WITHIN_MAP = True
BOOTSTRAP_RADIUS = 70
GIVE_UP_KNOWN = 75 
GIVE_UP_UNKNOWN = 60
SKIP_SPAWN = 90

**Comment by baskettcase** [comment]
--bootstrap seems to be doing better. it spread them out a lot more. Once it ran through fully for several hours and then I removed bootstrap they go back to congregating in one or two areas :( This seems to be a problem with the polygon.. the logic is built for rectangles?

**Comment by baskettcase** [comment]
No I just did a rectangle and they are all congregating in the same spot again.. and this time Im using bootstrap also. There is definitely some kind of problem with the logic.

**Comment by Noctem** [comment]
If you have `--bootstrap`ped with `BOOTSTRAP_RADIUS = 70` and let it run for a while, and there are still only 63 spawn points, then 16 workers is more than needed and clustering will be probable if spawn points are clustered.

**Comment by evenly-epic-mule** [comment]
https://github.com/Noctem/Monocle/pull/168 should fix this
